core/gui/src/app/workspace/service/workflow-graph/model/joint-graph-wrapper.ts

```diff
@@ -34,16 +34,13 @@ type JointLinkChangeEvent = [joint.dia.Link, { x: number; y: number }, { ui: boo
 
 type JointPositionChangeEvent = [joint.dia.Element, { x: number; y: number }];
 
-type JointLayerChangeEvent = [joint.dia.Element | joint.dia.Link, number];
```

Comment:
why remove this?

Reply:
After removing the operator group, we no longer have the OperatorLayer concept.